scripts/version-repo.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+# Copyright 2018 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+REPO_VER=v`cat ../CHANGELOG.md | grep "##" | head -n 1 | cut -d" " -f2`
```

Comment:
This assumes the script will be executed from the `scripts/` directory, which is not a good assumption IMO.